bin/ncp-update-nc

```diff
@@ -194,8 +194,8 @@ else
     apachectl -k graceful
     ## make sure the notify_push daemon is runnnig
 
-    arch="$(uname -m)"
-    [[ "${arch}" =~ "armv7" ]] && arch="armv7"
+    arch="$(dpkg --print-architecture)"
+    [[ "${arch}" = "armhf" ]] && arch="armv7"
```

Comment:
It looks good :) 
Tho there should be 2 equal signs inside the double-brackets, I've added a suggestion below which also uses the previous method together with dpkg
```suggestion
    [[ "$(uname -m)" =~ "armv7" ]] || [[ "${arch}" == "armhf" ]] && arch="armv7"
```